crates/cheatnet/tests/starknet/resources.rs

```diff
@@ -0,0 +1,60 @@
+use std::collections::HashMap;
```

Comment:
Can we add some more advanced cases, like with other builtins? 
This can be done in a separate PR

Reply:
#635